### ISSUE TITLE
So i want to optimize a mod but i dont know how to code, is anybody good at coding, or a video?

### ISSUE THREAD

**Issue by Tigedraglon10** [luckydog7/Funkin-android#1717]
Yeah I wanna optimize mods

**Comment by HatsuneMiku-github** [comment]
You don't need to code, should I do a tutorial?

**Comment by Pogsher** [comment]
No code needed afaik

**Comment by Pogsher** [comment]
> You don't need to code, should I do a tutorial?

Yes please. Im too lazy to wait for people to optimize camellia mod

**Comment by HatsuneMiku-github** [comment]
> > You don't need to code, should I do a tutorial?
> 
> Yes please. Im too lazy to wait for people to optimize camellia mod

Make a 1:1 resolution transparent ng image on ibis paint x and replace them with the assets you want to delete but you think it's gonna crash 

If you don't want to delete it and want it to stay extract the one you want to stay and go to ibis paint x don't press the lower resolution press the original one and then use the transform tool thingy and make it small and then save it and replace it with the original file

**Comment by HatsuneMiku-github** [comment]
Btw use apk editor

**Comment by DabKnight** [comment]
If you want to optimize but still see the sprites,watch recoloredpoyo's xml tutorial.
https://youtu.be/rGhDVZBaCH4
